Makefile

```diff
@@ -154,7 +154,7 @@ test-valgrind: $(LIBFLUX_MEMTEST_BIN)
 LIBFLUX_MEMTEST_SOURCES=libflux/c/*.c
 $(LIBFLUX_MEMTEST_BIN): libflux $(LIBFLUX_MEMTEST_SOURCES)
 	$(CC) -g -Wall -Werror $(LIBFLUX_MEMTEST_SOURCES) -I./libflux/include \
-		./libflux/target/debug/libflux.a ./libflux/target/debug/liblibstd.a \
```

Comment:
Ahh, this was the change to get rid of the second crate, right?

Reply:
Yup, that little patch I sent to the slack channel did that. It showed that the liblibsta.a file was already a combination of the two crates and in terms of linking in C, we already have them combined. This entire patch sort of makes it official, but renames them in the process.